css-canvas-medical/html/blog.html

```diff
@@ -0,0 +1,110 @@
+        <a href="index.html">
+          <img class="logo" src="../images/logo-blue.svg" alt="canvas" />
+        </a>
+      </div>
+      <nav>
+        <ul class="nav-bar column-two-thirds display-flex">
+          <li class="nav">
+            <a href="product.html" class="hyperlink"> Product</a>
+          </li>
+          <li class="nav">
+            <a href="implementation.html" class="hyperlink"> Implementation </a>
+          </li>
+          <li class="nav">
+            <a href="./about-us.html" class="hyperlink"> About Us </a>
+          </li>
+          <li class="nav">
+            <a href="#" class="hyperlink"> Blog </a>
+          </li>
+          <li class="nav">
+            <button class="contact-us">
+              <a href="contact-us.html" class="hyperlink-button">
+                Contact Us
+              </a>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </section>
+    <hr class="nav-bar-hr" />
+
+    <!-- Banner Section -->
+    <h1>Page coming soon...</h1>
+
+    <!-- Footer Section -->
+    <section class="display-flex">
+      <div class="learn-more-text column-half">
+        <h4 class="learn-more-title">Ready to learn more?</h4>
+        <p class="learn-more-textbox">Schedule a private meeting now</p>
+      </div>
+      <div class="learn-more-btn-container display-flex column-half">
+        <button class="learn-more-btn">Get Started</button>
+      </div>
+    </section>
+    <section class="info display-flex">
+      <div class="info-container column-half">
+        <img class="info-logo" src="../images/logo-white.png" alt="canvas" />
+        <div class="address">
+          <p>
+            2307 Irving Street, Suite 228
+            <br />
+            San Francisco CA 94122
+            <br />
+            (800) 370-1416
+            <br />
+            info@canvasmedical.com
+          </p>
+        </div>
+      </div>
+      <div class="info-container-2 column-half display-flex">
+        <div class="company column-half">
+          <p>COMPANY</p>
+          <ul class="info-list">
+            <li>
+              <a href="about-us.html" class="footer-link"> About Us </a>
+            </li>
+            <li><a href="blog.html" class="footer-link"> Blog </a></li>
+          </ul>
+        </div>
+        <div class="product column-half">
+          <p>PRODUCT</p>
+          <ul class="info-list">
+            <li>
+              <a href="product.html" class="footer-link"> Product </a>
+            </li>
+            <li>
+              <a href="implementation.html" class="footer-link">
+                Implementation
+              </a>
+            </li>
+            <li>For Health Plans</li>
+            <li>Security</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <footer class="display-flex">
+      <p class="terms">Privacy &amp; Terms</p>
+      <p class="contact">
+        <a href="contact-us.html" class="foot-link">Contact Us</a>
+      </p>
```

Comment:
Is there a reason this is wrapped in a `<p>`? If you were just doing it to make it block, I would've given the `<a>` `display: block` instead